src/main/steps/applicant1/number-of-applicants/content.ts

```diff
@@ -0,0 +1,63 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+
+const en = ({ required }) => ({
+  section: 'Applicant details',
+  title: 'Are you applying on your own, or with someone else?',
+  one: "I'm applying on my own",
+  two: "I'm applying with my spouse or civil partner",
+  three: "I'm applying with someone who is not my spouse or civil partner",
+  errors: {
+    noOfApplicants: {
+      required,
+    },
+  },
+});
+
+const cy = ({ required }) => ({
+  section: 'Applicant details (but in welsh)',
+  title: 'Are you applying on your own, or with someone else?(in welsh)',
+  one: "I'm applying on my own",
+  two: "I'm applying with my spouse or civil partner",
+  three: "I'm applying with someone who is not my spouse or civil partner",
+  errors: {
+    noOfApplicants: {
+      required,
+    },
+  },
+});
+
+export const form: FormContent = {
+  fields: {
+    noOfApplicants: {
+      type: 'radios',
+      classes: 'govuk-radios',
+      label: l => l.title,
+      section: l => l.section,
+      labelHidden: true,
+      values: [
+        { label: l => l.one, value: "I'm applying on my own" },
+        { label: l => l.two, value: "I'm applying with my spouse or civil partner" },
+        { label: l => l.three, value: "I'm applying with someone who is not my spouse or civil partner" },
```

Comment:
values should not be the label in this case, because we need to store it in CCD
can you change it to
`alone`
`withSpouseOrCivilPartner`
`withSomeoneElse`